src/components/WatchVideo.vue

```diff
@@ -144,6 +149,7 @@ import Comment from "@/components/Comment.vue";
 export default {
     name: "App",
     data() {
+        const smallViewQuery = window.matchMedia("(max-width: 640px)");
```

Comment:
Please move this to a computed variable.

Reply:
I tried as you suggested but I also had to initialize smallView to a boolean as it seems you can't access computed properties in data definition. Let me know if you had another way in mind.

Reply:
I had thought about removing `smallViewQuery` and replacing it with `smallView` as a computed variable!

Reply:
I tried that but it wouldn't update itself when I resized the window. I could use forceUpdate in the event listener instead though.

`    computed: {
        smallView() {
            return window.matchMedia("(max-width: 640px)").matches;
        }
    },`

Reply:
Oh okay, then your latest commit was unnecessary then, I'll revert it and merge this!